### ISSUE TITLE
Remap rshift

### ISSUE THREAD

**Issue by TheNoim** [koekeishiya/skhd#189]
Hey,

I try to remap the `rshift` key, because the macbook has awful small `arrow up` and `arrow down` keys. I want to remap `rshift` with `arrow up` and `arrow up` with `arrow down`. I never used the `rshift` key anyway, so why not remap it and get a bigger keypad. 

I tried 

```
rshift : skhd -k "up"
```

and 

```
0x3C : skhd -k "up"
```

The first one just results in `#5:10 expected '-'`. The second one in nothing. Is `0x3C` even the correct key code? `skhd -o` shows it when I press `rshift`.

**Comment by berenddeboer** [comment]
I don't think you can remap just modifier keys. The latest Mac OS allows you to do that via its settings somewhat better than before, but not as fine-grained as one would like.

**Comment by tjex** [comment]
> I don't think you can remap just modifier keys. The latest Mac OS allows you to do that via its settings somewhat better than before, but not as fine-grained as one would like.

dang, this put a nail further in the coffin for remapping caps lock to backspace :( 
+1 for remapping modifiers themselves - with more options than Apple gave us.
